src/server/views/index.pug

```diff
@@ -60,6 +60,55 @@ block content
     //-   p Learn the key concepts that run our interconnected world from iPhones through Instagram to traffic lights.
 
   .container-fluid#showcase
+    .row
```

Comment:
why didn't you just use a fn for this? there seems to be a bit of code duplication and might be annoying to change the featured projects?